Filter/StringFilter.php

```diff
@@ -55,7 +55,7 @@ public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $data)
      *
      * @return bool
      */
-    private function getOperator($type)
+    protected function getOperator($type)
```

Comment:
There is an even easier way than injection : `$proxyQuery->getEntityManager()->getConnection()->getDatabasePlatform()`. What do you think @Soullivaneuh @OskarStark ?